zng/docs/spec.md

```diff
@@ -514,39 +654,62 @@ This form defines an alias mapping the identifier to the indicated type.
 `<type-name>` is an identifier with semantics as defined in [Section 3.1.1.5](#3115-alias-typedef).
 
 
-### 4.1.3 Application-Specific Payload
+### 4.1.3 Application-Defined Messages
 
-A TZNG application-specific payload has the following form:
+A TZNG application-defined message has the following form:
 ```
-#!<control-code>:<payload>
+#!<control-code>:<message>
 ```
-Here, `<control-code>` is a decimal integer in the range 6-127 and `<payload>`
+Here, `<control-code>` is a decimal integer in the range 248-254 and `<message>`
 is any UTF-8 string with escaped newlines.
 
+### 4.1.4 End of Stream
+
+A TZNG end-of-stream marker has the following form:
+```
+#!255
+```
+A TZNG stream or file should always be terminated with end-of-stream.
+This clears all of the previous type tag bindings and aliases, allowing
+multiple TZNG files with overlapping tags to be concatenated without the
+tags colliding.
+
+> TBD: implement TZNG EOS in zq
```

Comment:
Should this just be moved to the top of the doc with the other zq "not implemented" exceptions? Or is it here because you intend to implement it really soon?

Reply:
Yes, also am changing this to `#eos`.